components/coding/studentPortal/Sidebar.tsx

```diff
@@ -35,6 +41,17 @@ export const Sidebar: React.FC<SidebarProps> = ({}: SidebarProps) => {
   const { signOut, user } = useAuth();
 
   const router = useRouter();
+  //commented out code (goalDateThreshold) is for a feature that will show the number of goals due within 7 days
+  //feature will be implemented soon.
+  // const goalDateThreshold = format(addDays(new Date(), 7), "MM/dd/yyyy");
+  // code commented out in fetchUserGoalsCount query as well.
+
+  const { data } = useQuery<FetchGoalCountResponse>(FETCH_USER_GOALS_COUNT, {
```

Comment:
let's actually comment this portion out because it will run every time skillify loads and we aren't using it yet